usaspending_api/awards/models.py

```diff
@@ -494,29 +498,27 @@ class Meta:
 
 
 class SubAward(DataSourceTrackedModel):
```

Comment:
Very minor point, but what do you think about un-CamelCasing this for future ease of typing?